helm/charts/goquorum-node/templates/node-service.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     app.kubernetes.io/component: {{ .Release.Name }}
   namespace: {{ .Release.Namespace }}
 spec:
-  type: ClusterIP
+  type: {{ .Values.node.goquorum.serviceType }}
```

Comment:
Please specify nodeport field to have a static port of `rlpx`, and `discovery` protocol, which used in static-nodes.config